### DISCUSSION TITLE
VCFdata throwing a 'nil value'

### DISCUSSION THREAD

**Discussion by Ninjaborn** [matsn0w/IIS-EmergencyLS-ELS-FiveM#73] [Q&A]
Hey, so attempting to use this on our server and after installing and getting all the vehicles converted over, I'm getting an error when someone else turns on their lightbar that states VCFdata is indexing a nil value. Can't really spot where the error is originating from however as VCFdata is defined?

https://gyazo.com/ce21b936a736165f0cbb0f97ef8b6563

Much appreciated any help you can give!

**Comment by matsn0w** [comment]
Hey, how did you download the resource? You have 'MISS-ELS-main' in your script name, so I'm suspecting you have downloaded the entire repo instead of just the release artifact. Can you please try again with the latest release? 

[MISS-ELS_v2.0.1.zip](https://github.com/matsn0w/MISS-ELS/releases/download/2.0.1/MISS-ELS_v2.0.1.zip)

**Reply by Ninjaborn** [reply]
Thanks for the reply. I did download the entire repo however, I replaced the contents with the updated artifact after reading through your github but never renamed it so that's my mistake.

I did try using the exact file you sent here as well but am still receiving that nil value for VCFdata. Not sure where the problem lies as the xml files work on the clients side, but other users are seeing an incorrect light pattern on the server.

**Reply by Ninjaborn** [reply]
So looking at the issues, I think we are experiencing something similar to issue 41. As you stated at the time is not fixable. We will attempt to look into this as well but I do thank you for your time.

**Reply by FBFezz** [reply]
Having the same issue

**Reply by oiChips** [reply]
Having the same issue. Happens whenever lights are activated. Only happens when somebody else turns lights on. 

If person A is standing outside of the ELS car and person B activates the ELS, Person A will see the error in F8 console. 

Would really appreciate a potential fix for this. 

Thanks

**Reply by matsn0w** [reply]
Ok, thank you all for your reports. Will have a look into this!

**Reply by oiChips** [reply]
Do you have any further details on this? I really want to use this ELS but its spamming hell out of everyones console
